src/main/kotlin/snyk/container/ui/BaseImageRemediationDetailPanel.kt

```diff
@@ -0,0 +1,292 @@
+    }
+
+    private fun baseRemediationInfoPanel(): JPanel {
+        val panel = JPanel(GridLayoutManager(4, 4, Insets(20, 0, 0, 0), 0, 10))
+
+        panel.add(
+            JLabel("Recommendations for upgrading the base image"),
+            baseGridConstraints(0)
+        )
+
+        panel.add(
+            innerRemediationInfoPanel(),
+            baseGridConstraints(1)
+        )
+
+        return panel
+    }
+
+    private fun innerRemediationInfoPanel(): JPanel {
+        val panel = JPanel(GridLayoutManager(3, 4, Insets(0, 0, 0, 0), 0, 10))
+
+        imageIssues.baseImageRemediationInfo?.currentImage?.let {
+            addBaseImageInfo(panel, 0, CURRENT_IMAGE, it)
+        }
+
+        imageIssues.baseImageRemediationInfo?.minorUpgrades?.let {
+            addBaseImageInfo(panel, 1, MINOR_UPGRADES, it)
+        }
+
+        imageIssues.baseImageRemediationInfo?.alternativeUpgrades?.let {
+            addBaseImageInfo(panel, 2, ALTERNATIVE_UPGRADES, it)
+        }
+
+        return panel
+    }
+
+    private fun addBaseImageInfo(panel: JPanel, row: Int, title: String, info: BaseImageInfo) {
+        panel.add(
+            JLabel(title).apply {
+                font = io.snyk.plugin.ui.getFont(Font.BOLD, -1, JLabel().font)
+            },
+            baseGridConstraints(row, 0, indent = 0)
+        )
+        panel.add(
+            JLabel(info.name).apply {
+                name = title
+            },
+            baseGridConstraints(row, 1, indent = 5)
+        )
+        panel.add(
+            getVulnerabilitiesCHMLpanel(info.vulnerabilities),
+            baseGridConstraints(row, 2, indent = 5)
+        )
+    }
+
+    private fun panelGridConstraints(row: Int) = baseGridConstraints(
+        row = row,
+        anchor = GridConstraints.ANCHOR_CENTER,
+        fill = GridConstraints.FILL_BOTH,
+        HSizePolicy = GridConstraints.SIZEPOLICY_CAN_SHRINK or GridConstraints.SIZEPOLICY_CAN_GROW,
+        VSizePolicy = GridConstraints.SIZEPOLICY_CAN_SHRINK or GridConstraints.SIZEPOLICY_CAN_GROW,
+        indent = 0
+    )
+
+    private fun baseGridConstraints(
+        row: Int,
+        column: Int = 0,
+        rowSpan: Int = 1,
+        colSpan: Int = 1,
+        anchor: Int = GridConstraints.ANCHOR_WEST,
+        fill: Int = GridConstraints.FILL_NONE,
+        HSizePolicy: Int = GridConstraints.SIZEPOLICY_FIXED,
+        VSizePolicy: Int = GridConstraints.SIZEPOLICY_FIXED,
+        minimumSize: Dimension? = null,
+        preferredSize: Dimension? = null,
+        maximumSize: Dimension? = null,
+        indent: Int = 1,
+        useParentLayout: Boolean = false
+    ): GridConstraints {
+        return GridConstraints(
+            row, column, rowSpan, colSpan, anchor, fill, HSizePolicy, VSizePolicy, minimumSize, preferredSize,
+            maximumSize, indent, useParentLayout
+        )
+    }
+
+    private fun getVulnerabilitiesCHMLpanel(vulnerabilities: BaseImageVulnerabilities?): JPanel {
```

Comment:
What does CHMLpanel stand for? Also, Panel should probably be capitalized ;).

Reply:
CHML - Critical High Medium Low
also having "panel" capitalized in proper camelCase will interfere with CHML abbreviation here.